seeds/friday_projects.js

```diff
@@ -0,0 +1,32 @@
+exports.seed = function (knex, Promise) {
+  return Promise.all([
+    knex('projects').del(),
+    knex('photos').del(),
+    knex('team_members').del(),
+    knex('project_teams').del()
+  ])
+    .then(function () {
+      return knex('projects').insert({id: 1, title: 'Project One', description: 'This is project one', repo_url: 'http://github.com/projectone', app_url: 'http://projectone.com', date: '02/08/2016'})
+    })
+    .then(function () {
+      return knex('projects').insert({id: 2, title: 'Project Two', description: 'This is project two', repo_url: 'http://github.com/projecttwo', app_url: 'http://projecttwo.com', date: '03/08/2016'})
+    })
+    .then(function () {
+      return knex('photos').insert({id: 1, url: 'http://cache.gawkerassets.com/assets/images/17/2010/05/500x_afternoon-projects.jpg', caption: 'what a cool photo', projects_id: 1})
```

Comment:
`projects_id` should be renamed to `project_id` (on the next line too).

Another idea would be to put each of the properties on their own line so the line isn't so long. Not really important. Do what you think is best.